spec/project_spec.rb

```diff
@@ -70,8 +71,9 @@ def expected
 
     it 'has a period at EOL of description' do
       cop_names.each do |name|
-        description = config[name]['Description']
+        next unless config[name]
 
+        description = config[name]['Description']
```

Comment:
Others have been replaced with `dig`, is there any reason why this was not replaced?

Reply:
Because there's no point in having two tests fail for there being no description IMO. The description one will fail if there is no description and this one will be skipped.